Objects/Converters/ConverterRevit/ConverterRevitShared/PartialClasses/ConvertCeiling.cs

```diff
@@ -126,7 +125,7 @@ public ApplicationObject CeilingToNative(Ceiling speckleCeiling)
     {
       CreateVoids(revitCeiling, speckleCeiling);
     }
-    catch (Exception ex)
+    catch (Autodesk.Revit.Exceptions.ApplicationException ex)
     {
       appObj.Update(logItem: $"Could not create openings: {ex.Message}");
```

Comment:
If we fail to create something, shouldn't we also update the status of the appObj to be failed?